pkg/registry/node/runtimeclass/strategy.go

```diff
@@ -58,12 +56,7 @@ func (strategy) AllowCreateOnUpdate() bool {
 // PrepareForCreate clears fields that are not allowed to be set by end users
 // on creation.
 func (strategy) PrepareForCreate(ctx context.Context, obj runtime.Object) {
-	rc := obj.(*node.RuntimeClass)
-
-	if !utilfeature.DefaultFeatureGate.Enabled(features.PodOverhead) && rc != nil {
-		// Set Overhead to nil only if the feature is disabled and it is not used
-		rc.Overhead = nil
-	}
+	_ = obj.(*node.RuntimeClass)
```

Comment:
no need to keep this line, can be an empty method

Reply:
Updated